Test/dafny3/Inc.dfy.expect

```diff
@@ -8,7 +8,7 @@ Inc.dfy(83,12): Related location: This is the postcondition that might not hold.
 Execution trace:
     (0,0): anon0
     (0,0): anon3_Then
-Inc.dfy(93,2): Error BP5003: A postcondition might not hold on this return path.
+Inc.dfy(92,8): Error BP5003: A postcondition might not hold on this return path.
```

Comment:
Is the more specific information about which `case` is on the failing path no longer available?